### PR TITLE
22803 - Fix 65535+ query parameters error

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov/entity/issues/22803

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: jobs/payment-jobs/tests/jobs/test_generate_statements.py [hunk 1]
```diff
@@ -274,6 +275,38 @@ def test_bcol_monthly_to_eft_statement(session):
...
+    invoice = factory_invoice(account)
+    statement_list = []
+    for _ in range(0, 70000):
+        statement_list.append({'created_on': '2024-01-01',
+                               'from_date': '2024-01-01 08:00:00',
+                               'to_date': '2024-01-04 08:00:00',
+                               'payment_account_id': f'{account.id}',
+                               'frequency': StatementFrequency.DAILY.value
+                               })
+    db.session.execute(insert(Statement), statement_list)
```
Comment (seeker25) [inline jobs/payment-jobs/tests/jobs/test_generate_statements.py:296]: faster suggested way than bulk insert according to docs

File: jobs/payment-jobs/tests/jobs/test_generate_statements.py [hunk 2]
```diff
@@ -274,6 +275,38 @@ def test_bcol_monthly_to_eft_statement(session):
...
+    statement = db.session.query(Statement).first()
+    statement_invoices_list = []
+    for _ in range(0, 70000):
+        statement_invoices_list.append({
+            'statement_id': statement.id,
+            'invoice_id': invoice.id
+        })
+    db.session.execute(insert(StatementInvoices), statement_invoices_list)
+    StatementTask.generate_statements([datetime(2024, 1, 1, 8).strftime('%Y-%m-%d')])
+    assert True
```
Comment (seeker25) [inline jobs/payment-jobs/tests/jobs/test_generate_statements.py:307]: Usually throws exception before this

**Comment by seeker25** [comment]
A better way of fixing this problem, I think would be rewriting it so we pass in the query object instead of a hard id list

**Review by ochiu — APPROVED** [review]
looks good to me

**Pull request merged**